src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IForEachLoopStatement.cs

```diff
@@ -1219,12 +1220,13 @@ public static void M(int[] x)
 IBlockOperation (1 statements) (OperationKind.Block, Type: null, IsInvalid) (Syntax: '{ ... }')
   IForEachLoopOperation (LoopKind.ForEach) (OperationKind.Loop, Type: null, IsInvalid) (Syntax: 'foreach (x[ ... }')
     LoopControlVariable: 
```

Comment:
>LoopControlVariable [](start = 4, length = 19)

It looks like now IForEachLoopStatement.LoopControlVariable should never be null, please make necessary adjustments (add/change asserts in TestOperationVisitor and OperationTreeVerifier)